packages/slice-machine/components/Simulator/components/FailedConnect/index.tsx

```diff
@@ -17,8 +17,7 @@ const FailedConnect = () => (
     </Text>
     <Text
       sx={{
-        color: "#86848D",
-        fontWeight: "400",
+        color: "failedConnectText",
```

Comment:
I don't think that's the name used in the design system.

Reply:
hmm yeah I called it like that because it's only used there